client/client_test.go

```diff
@@ -90,9 +90,18 @@ func TestAPI(t *testing.T) {
 	client := &fakeAPIClient{T: t, ch: make(chan fakeAPIResponse, 1)}
 	now := time.Now()
 
+	u, err := url.Parse("http://example.com")
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
 	statusData := &ServerStatus{
-		ConfigYAML:    "{}",
-		ConfigJSON:    &config.Config{},
+		ConfigYAML: "{}",
+		ConfigJSON: &config.Config{
+			Global: &config.GlobalConfig{
+				PagerdutyURL:  &config.URL{u},
```

Comment:
`composites: `github.com/prometheus/alertmanager/config.URL` composite literal uses unkeyed fields (govet)`

Reply:
I would guess this means `u` needs to be given a key, but .. never seen this error before so :)

Reply:
Exactly!